addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -5020,29 +5097,41 @@ registry.BackgroundToggler = SnippetOptionWidget.extend({
         return shapeOption._toggleShape();
     },
     /**
-     * Toggles background filter on or off.
+     * Removes remaining transparent bg filter
      *
-     * @see this.selectClass for parameters
+     * @override
      */
-    toggleBgFilter(previewMode, widgetValue, params) {
-        if (widgetValue) {
-            const bgFilterEl = document.createElement('div');
-            bgFilterEl.classList.add('o_we_bg_filter', 'bg-black-50');
-            const lastBackgroundEl = this._getLastPreFilterLayerElement();
-            if (lastBackgroundEl) {
-                $(lastBackgroundEl).after(bgFilterEl);
-            } else {
-                this.$target.prepend(bgFilterEl);
+    async cleanForSave() {
+        const filter = this.$target.find('.o_we_bg_filter');
+        if (filter) {
```

Comment:
This condition is always true, this is not python :smile: An empty object is truthy.